src/test/java/UITest.java

```diff
@@ -1,10 +1,10 @@
+import org.hamcrest.CoreMatchers;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.*;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
It usually depends on the team that you are on, but I like to prevent IntelliJ from using * imports.  

`Settings > Editor > Code Style > Java > Imports > Class count to use import with '*'`